src/pages/contact_us/_offices.tsx

```diff
@@ -230,17 +230,17 @@ export const Offices = () => {
                                     mt="0.8rem"
                                     ml="1.6rem"
                                 >
-                                    <Show.Desktop min_width="mobileL">
+                                    <Desktop breakpoint={size.mobileL}>
                                         <p>Level 3, W Business Centre,</p>
                                         <p>Triq Dun Karm, Birkirkara,</p>
                                         <p>BKR 9033</p>
-                                    </Show.Desktop>
-                                    <Show.Mobile>
+                                    </Desktop>
+                                    <Mobile>
```

Comment:
Do you not also need breakpoint for mobile?

Reply:
In the original code, mobile component is not taking any props such as min or max width, so I assumed breakpoint is not a requirement for mobile.

Reply:
Please do go through the `visibility.tsx` file and try to read through the logic written there. The breakpoints should match for both desktop and mobile to avoid unnecessary display of the mobile contents during certain screen widths.